test/mail_dnsbl.bats

```diff
@@ -0,0 +1,64 @@
+load 'test_helper/common'
+
+function setup() {
+    run_setup_file_if_necessary
+}
+
+function setup_file() {
+  local PRIVATE_CONFIG CONTAINER CONTAINER2
+  PRIVATE_CONFIG="$(duplicate_config_for_container . "${CONTAINER}")"
```

Comment:
Does this actually work? `CONTAINER` is declared local var prior but no value assigned, only after this line do you set `CONTAINER` to a string... shouldn't that come before defining `PRIVATE_CONFIG`?

Reply:
This is fine and works. I never saw it the other way. Any example?

Reply:
> Any example?

The lines I've commented on here, and the ones that follow directly after. Is the shell script hoisting or is `${CONTAINER}` at this point empty and misleading in usage?

```bash
  local PRIVATE_CONFIG CONTAINER CONTAINER2
  PRIVATE_CONFIG="$(duplicate_config_for_container . "${CONTAINER}")"
  CONTAINER="mail_dnsbl_enabled"
  CONTAINER2="mail_dnsbl_disabled"
```

Reply:
Now I get it! 😆 

I thought you meant something like this, which confused me:

```
PRIVATE_CONFIG="$(duplicate_config_for_container . "${CONTAINER}")"
local PRIVATE_CONFIG CONTAINER CONTAINER2
```

You are right. CONTAINER should be empty at this point 😞 I'll fix this. Hopefully the last fix in this PR 😆

Reply:
Fixed with the (hopefully) last commit for that PR 😄